env.js

```diff
@@ -97,5 +97,7 @@ module.exports = (function() {
   // The local host to publish to discovery
   env.publishHost = config.fromEnvironment('PUBLISH_HOST');
 
+  env.skipHakken = config.fromEnvironment('SKIP_HAKKEN', false);
```

Comment:
This should either be `env.discovery.skipHakken`, or be included up where `env.discovery` is defined.